Tests/NIOSSLTests/TLSConfigurationTest.swift

```diff
@@ -401,4 +401,44 @@ class TLSConfigurationTest: XCTestCase {
         XCTAssertEqual(config.applicationProtocols, ["h2", "http/1.1"])
         XCTAssertEqual(config.encodedApplicationProtocols, [[2, 104, 50], [8, 104, 116, 116, 112, 47, 49, 46, 49]])
     }
+
+    func testKeyLogManagerOverlappingAccess() throws {
+        // Tests that we can have overlapping calls to the log() function of the keylog manager.
+        // This test fails probabilistically! DO NOT IGNORE INTERMITTENT FAILURES OF THIS TEST.
```

Comment:
Did this fail frequently enough to warrant running it in a loop a handful of times? Bit of a bummer we need the 10ms wait though.

Reply:
I don't think so, tbh. The test itself is constructed such that the act of fixing it made the failing version of the test impossible to write (`log` became non-`mutating`), so it's a bit odd anyway.

Reply:
Oh of course, I didn't register the `mutating` being dropped.